modules/menu/menu.c

```diff
@@ -134,6 +134,14 @@ static char *errorcode_key_aufile(uint16_t scode)
 }
 
 
+static void menu_stop_play(void)
+{
+	menu.play = mem_deref(menu.play);
+	menu.ringback = false;
+	alert_stop();
```

Comment:
unrelated comment; should we consider to remove the "alert" code ?

Reply:
Yes, I think so. Another PR or may add here a commit?